From dc56b9e3d24386dd7e74023fe7adecbbee64aa4c Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Wed, 20 Jan 2021 11:14:58 +0100
Subject: [PATCH 111/841] feat(fdts): add RISAF nodes for STM32MP25

Add RISAF2 and RISAF4 nodes in STM32MP25 SoC DT file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 4df16b2b4f6827b394e5d7cbff8c61476d92332e feat(fdts): add RISAF nodes for STM32MP25]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I96f96267292e7f7a498a87c60e35310e25b41d6d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/250862
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
---
 fdts/stm32mp251.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index f55a3b973..de59fd7a6 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -99,6 +99,20 @@
 			};
 		};
 
+		risaf2: risaf@420b0000 {
+			compatible = "st,stm32-risaf";
+			reg = <0x420b0000 0x1000>;
+			clocks = <&rcc CK_KER_OSPI1>;
+			status = "disabled";
+		};
+
+		risaf4: risaf@420d0000 {
+			compatible = "st,stm32-risaf";
+			reg = <0x420d0000 0x1000>;
+			clocks = <&rcc CK_BUS_RISAF4>;
+			status = "disabled";
+		};
+
 		rcc: rcc@44200000 {
 			compatible = "st,stm32mp25-rcc";
 			reg = <0x44200000 0x10000>;
-- 
2.47.2

